www/content/_redirects

```diff
@@ -1 +1,2 @@
-/* /404.html 404
\ No newline at end of file
+/fairuse https://mitocw.zendesk.com/hc/en-us/articles/4414756181403-How-is-all-rights-reserved-content-different-from-the-rest-of-OCW-content- 301
```

Comment:
For simple redirects like this we should be adding an entry to the file [here](https://github.com/mitodl/ol-infrastructure/blob/main/src/ol_infrastructure/applications/ocw_site/redirect_dict#L463)

We are using Fastly for managing redirects to avoid cluttering Hugo with that knowledge. I also believe that we had experimented with using this feature previously without much success due to how we are deploying the site.

Reply:
Thanks Tobias